<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="SendgroupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>SendData</string>
    </property>
    <widget class="QLineEdit" name="DataInput">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="HexSelect">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>十六进制</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>42</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>DataInput</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendData_Button">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ReceiveaData">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>421</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>RecData</string>
    </property>
    <widget class="QTextBrowser" name="DataRecDisplay">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>341</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ParamSet">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>191</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>ParamSet</string>
    </property>
    <widget class="QPushButton" name="ConnectButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UartConfigButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>UartConfig</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>647</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="Save"/>
    <addaction name="Exit"/>
   </widget>
   <widget class="QMenu" name="Communication">
    <property name="title">
     <string>通讯</string>
    </property>
    <addaction name="disconnect"/>
    <addaction name="connect"/>
    <addaction name="actionConfig"/>
   </widget>
   <widget class="QMenu" name="Help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="helpprotocol"/>
    <addaction name="helpabout"/>
    <addaction name="helpcontact"/>
   </widget>
   <addaction name="File"/>
   <addaction name="Communication"/>
   <addaction name="Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Open">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="Save">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="connect">
   <property name="text">
    <string>connect</string>
   </property>
  </action>
  <action name="disconnect">
   <property name="text">
    <string>disconnect</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="helpprotocol">
   <property name="text">
    <string>protocol</string>
   </property>
  </action>
  <action name="helpabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="Exit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="text">
    <string>Config</string>
   </property>
  </action>
  <action name="helpcontact">
   <property name="text">
    <string>contact</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
